orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/manifests/BakeManifestContext.java

```diff
@@ -33,17 +32,17 @@
   private final String outputName;
   private final String namespace;
 
-  // There does not seem to be a way to auto-generate a constructor using our current version of Lombok (1.16.20) that
+  // There does not seem to be a way to auto-generate a constructor using our current version of
+  // Lombok (1.16.20) that
```

Comment:
but we're on 1.18.x now, still relevant?

Reply:
No idea. Don't have the appetite to tackle it now.